CosmicReco/src/PDFFit.cc

```diff
@@ -348,14 +352,15 @@ double GaussianDriftFit::DOCAresidual(ComboHit const& sh, const std::vector<doub
   TwoLinePCA pca(intercept, dir, straw.getMidPoint(), straw.getDirection());
   double traj_time = ((pca.point1() - intercept).dot(dir)) / 299.9;
 
-  double predictedDistance = pca.dca();
+  double predictedDistance = HitAmbiguity(sh, x) * pca.dca();
```

Comment:
We don't want HitAmbiguity here anymore either right?

Reply:
I forgot to change the convention here as well. fixed now